nionswift_plugin/nion_experimental_4dtools/ComputationUI.py

```diff
@@ -32,6 +35,16 @@ def __update_computation_ui(self, computations):
         self.column._widget.remove_all()
         for computation in computations:
             create_panel_widget = getattr(computation, 'create_panel_widget', None)
+            if create_panel_widget is None:
+                compute_class = Symbolic._computation_types.get(computation.processing_id)
+                if compute_class:
+                    api_computation = self.api._new_api_object(computation)
+                    api_computation.api = self.api
+                    try:
+                        compute_class(api_computation)
+                    except IndexError:
+                        pass
+                    create_panel_widget = getattr(computation, 'create_panel_widget', None)
```

Comment:
What's the purpose of this section? (the reason I'm asking is if we don't need it, then the only change should be the  lambda function below -- and with appropriate arguments to that lambda, we can make this code compatible with both 0.14 and 0.15 for the time being...?

Reply:
I forgot about this part in my pull request. The purpose is to show the computation UI after a re-start of Swift.
The `create_panel_widget` attribute gets set in the `__init__` method of the computation class, so we need to instantiate it before we can access it. After restarting Swift, computations will only be instantiated when something in them changes, but for some of them you cannot change anything without the UI, so they would never get instantiated.